add prisma postgres to hyperdrive (draft, needs configurable connecti… (#23373)
cloudflare/cloudflare-docs: src/content/docs/workers/tutorials/using-prisma-postgres-with-workers/index.mdx

  - PostgreSQL
---

import { PackageManagers, Tabs, TabItem, Render } from "~/components";

[Prisma Postgres](https://www.prisma.io/postgres) is a managed, serverless PostgreSQL database. It supports features like connection pooling, caching, real-time subscriptions, and query optimization recommendations.

:::note

> You can connect to Prisma Postgres using Prisma Accelerate or [Hyperdrive](/hyperdrive). Both approaches provide comparable performance with different tradeoffs.

Author: ```suggestion
You can connect to Prisma Postgres using [Prisma Accelerate](https://www.prisma.io/accelerate) or [Hyperdrive](/hyperdrive). Both approaches provide comparable performance with different tradeoffs.
```

> Prisma Accelerate provides connection pooling and caching specifically designed for connecting to Prisma Postgres with the Prisma ORM, improving database access from Workers with built-in optimizations for the Prisma ecosystem.

Author: ```suggestion
Prisma Accelerate provides connection pooling and caching specifically designed for connecting to Prisma Postgres with the [Prisma ORM](https://www.prisma.io/), improving database access from Workers with built-in optimizations for the Prisma ecosystem.
```

> Hyperdrive provides the lowest possible latencies due to its globally distributed nature and allows you to use any standard PostgreSQL driver. Hyperdrive performs connection pooling across Cloudflare's network and supports native database drivers, libraries, and ORMs. Learn more about Hyperdrive in [How Hyperdrive Works](/hyperdrive/configuration/how-hyperdrive-works/).

Author: ```suggestion
Hyperdrive provides the lowest possible latencies when connecting to external databases from Workers, allows you to use any standard PostgreSQL driver, and is included in all Workers plans. Hyperdrive performs connection pooling across Cloudflare's network and supports native database drivers, libraries, and ORMs. Learn more about Hyperdrive in [How Hyperdrive Works](/hyperdrive/configuration/how-hyperdrive-works/).
```
